Strategy

Tiranu91 opened the pull request.

Tiranu91 committed:
Create strategy.md

Tiranu91 committed:
Update strategy.md

bastelfreak commented on strategy.md:
```diff
@@ -0,0 +1,58 @@
+# Strategy Pattern
+
+## Definition
+Das Strategypattern ermöglicht es, den Algorithmus unabhängig von ihn nutzenden Klienten zu variieren.
```
bitte leerzeile zwischen der überschrift und den textblöcken einfügen. Ein markdown validator ist in der aktuellen README.md verlinkt.

Tiranu91 committed:
Update strategy.md

- Leerzeichen zwischen Überschriften und Text
- Tippfehler in Codebeispiel korrigiert
- Codebeispiele werden die Tage überarbeitet

bastelfreak committed:
replace hard tabs
